config.json

```diff
@@ -332,6 +332,22 @@
         "structs"
       ]
     },
+    {
+      "slug": "pythagorean-triplet",
+      "uuid": "7b7c61f4-2849-46b7-9057-900bb6b95052",
+      "core": true,
+      "auto_approve": true,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
+        "control_flow_if_else_statements",
+        "control_flow_loops",
+        "memory_management",
+        "arrays",
+        "flexible_array_members",
+        "structs"
+      ]
+    },
```

Comment:
Not at all sure about the sequence of this within the overall tree 🤷‍♂

Reply:
:man_shrugging: